server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js

```diff
@@ -43,7 +43,6 @@ BuildOutputObserver.prototype = {
 
         if (this.is_completed && this.is_output_empty) {
             this.update_page(json);
-            dashboard_periodical_executer.pause();
```

Comment:
Why did you get rid of this call? Won't it start making calls to update job status of a completed build?